### ISSUE TITLE
TypeError when importing Python module or using stocks/qa in terminal [Bug]

### ISSUE THREAD

**Issue by DiGyt** [OpenBB-finance/OpenBBTerminal#2189]
**Describe the bug**
A wrong type description is causing type errors in [/stocks/quantitative_analysis/beta_model](https://github.com/OpenBB-finance/OpenBBTerminal/blob/a768e545a8c86ce8372cda1a41f62d49ce4fbe2b/openbb_terminal/stocks/quantitative_analysis/beta_model.py#L11) either when importing the python module or when using the qa from terminla.

**To Reproduce**
In Python:
```python
from openbb_terminal import api as openbb
```

> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/dirk/Programs/OpenBBTerminal/openbb_terminal/api.py", line 4, in <module>
    from .stocks import stocks_api as stocks
  File "/home/dirk/Programs/OpenBBTerminal/openbb_terminal/stocks/stocks_api.py", line 22, in <module>
    from .quantitative_analysis import qa_api as qa
  File "/home/dirk/Programs/OpenBBTerminal/openbb_terminal/stocks/quantitative_analysis/qa_api.py", line 52, in <module>
    models = _models(
  File "/home/dirk/Programs/OpenBBTerminal/openbb_terminal/helper_classes.py", line 85, in __init__
    loader.exec_module(getattr(self, model_name))
  File "/home/dirk/Programs/OpenBBTerminal/openbb_terminal/stocks/quantitative_analysis/beta_model.py", line 11, in <module>
    ) -> tuple[pd.Series, pd.Series, float, float]:
TypeError: 'type' object is not subscriptable

In Terminal:
```bash
python terminal.py
stocks
load AAPL
qa
```

> Error: 'type' object is not subscriptable

**Desktop (please complete the following information):**

- OS: Ubuntu 22.04 LTS
- Python version 3.8.13
- All libraries were freshly installed according to the [installation guide using Anaconda](https://github.com/OpenBB-finance/OpenBBTerminal/blob/d08501894f45dd60a0c3a3e9f158c2f23032776e/openbb_terminal/README.md)

Bugfix will be provided

**Comment by deeleeramone** [comment]
Hello,

The OpenBB API currently only works when the repo has been cloned and has been installed via a Conda virtual environment. The installer packages are not compatible. 

See the instructions for installing here: [https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/openbb_terminal/README.md#anaconda--python](https://github.com/OpenBB-finance/OpenBBTerminal/blob/main/openbb_terminal/README.md#anaconda--python
)

**Comment by DiGyt** [comment]
Hey @deeleeramone . Yes, that was exactly the way I installed it. I did a full install exactly executing the commands listed in the file, and as a result got the error mentioned above. I believe the line that caused the error is also just 2 days old, and the error isn't directly raised when opening the terminal (only when using stocks/quantitative_analysis or importing the module in Python). So I thought it either is a bug that snuck in the last days, or if that's not possible it might have to do with my Python version.

**Comment by deeleeramone** [comment]
I cannot recreate this error (Mac OS). Are you certain that the Python interpreter is set to the `obb` environment when interacting with the API? 

Did you also update your local system?

```
sudo apt update && sudo apt upgrade
sudo apt install -y cmake
pip install cmake
```

![Screen Shot 2022-07-28 at 8 09 47 AM](https://user-images.githubusercontent.com/85772166/181572995-70c447ce-1139-4599-b55a-d0847c7cc9ba.png)


<img width="1429" alt="Screen Shot 2022-07-28 at 8 06 39 AM" src="https://user-images.githubusercontent.com/85772166/181572282-b5023754-98be-4fa0-b89e-ac256bb7c292.png">

**Comment by DiGyt** [comment]
Hey @deeleeramone ,

Thanks for double checking. I did update my local system using
```
sudo apt update && sudo apt upgrade
sudo apt install -y cmake
pip install cmake
```
and the error still occurs.

Since it worked on your Python 3.10 version and did not on my Python 3.8 I checked the What's New summary of Python releases and found that there was a change in accepting tuple typing between Python 3.8 and Python 3.9:

Python 3.8:
```python
>>> tuple[int, int]
```

> Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'type' object is not subscriptable

Python 3.9:
```python
>>> tuple[int, int]
```

> \>>>

So I guess this problem occurs only for Python versions < 3.9 and can be solved by either setting the minimum Python version in the requirements to `python_version >= 3.9` instead of `python_version >= 3.5` or by changing the tuple instance to typing.Tuple as described in PR #2190 (to my knowledge the occurrence in stocks.quantitative_analysis is the only one in the entire repo).
